src/networking/MatterStatusService.ts

```diff
@@ -0,0 +1,48 @@
+import ModelService from "./ModelService";
+import { where, orderBy, doc } from "@firebase/firestore";
+import {
+  COLLECTION_NAME,
+  Populate,
+  PopulationOptions,
+  REF_PROPERTY_NAME,
+} from "./PopulationOptions";
+import { ORDER_DIRECTION, WHERE_OPERATOR } from "./constants";
+import { NetworkService } from "./NetworkService";
+
+import MatterStatus from "../models/MatterStatus";
+import { FirebaseConfig } from "../app/AppConfigContext";
+
+export default class MatterStatusService extends ModelService {
+  constructor(firebaseConfig: FirebaseConfig) {
+    super(COLLECTION_NAME.MatterStatus, firebaseConfig);
+  }
+
+  async getMatterStatusesByMatterId(matterId: string): Promise<MatterStatus[]> {
+    const populateMatter = new Populate(
+      COLLECTION_NAME.MatterStatus,
+      REF_PROPERTY_NAME.MatterStatusMatterRef
+    );
```

Comment:
We don't need to populate `matter_ref` because they are all the same matter anyways.

In the list of matter statuses, I don't think the full `matter` is ever used?